presto-hive/src/test/java/io/prestosql/plugin/hive/TestCoralIntegration.java

```diff
@@ -0,0 +1,136 @@
+        queryRunner.createCatalog("tpch", "tpch", ImmutableMap.of());
+
+        // Create hive catalog
+        FileHiveMetastore metastore = FileHiveMetastore.createTestingFileHiveMetastore(catalogDirectory);
+        queryRunner.installPlugin(new TestingHivePlugin(metastore));
+        queryRunner.createCatalog(HIVE_CATALOG, "hive", ImmutableMap.of());
+
+        // Populate TPC-H tables to Hive
+        metastore.createDatabase(new HiveIdentity(session.toConnectorSession()), new Database(TPCH_SCHEMA, Optional.empty(), "ignored", PrincipalType.USER, Optional.empty(), ImmutableMap.of()));
+        Session tpchSession = Session.builder(queryRunner.getDefaultSession()).setSchema("tpch").build();
+        copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, tpchSession, ImmutableList.of(TpchTable.ORDERS));
+
+        // And create the Hive-defined views we'll use for testing
+        createHiveView(
+                metastore,
+                new HiveIdentity(session.toConnectorSession()),
+                "tpch",
+                "counting_view",
+                "select count(1) as count from tpch.orders",
+                "select count(1) as `count` from `tpch`.`orders`");
+        createHiveView(
+                metastore,
+                new HiveIdentity(session.toConnectorSession()),
+                "tpch",
+                "zero_index_view",
+                // Arrays in Hive are zero-indexed. This query will return 'hive' only if it's being parsed as a Hive query
+                "select array('presto','hive')[1] as sql_dialect",
+                "select array('presto','hive')[1] as `sql_dialect`");
+
+        return queryRunner;
+    }
+
+    private static void createHiveView(FileHiveMetastore metastore, HiveIdentity identity, String databaseName, String tableName, String originalViewText, String expandedViewText)
+    {
+        Storage storage = Storage.builder()
+                .setLocation("")
+                .setStorageFormat(StorageFormat.VIEW_STORAGE_FORMAT)
+                .setBucketProperty(Optional.empty())
+                .setSerdeParameters(ImmutableMap.of())
+                .build();
+        metastore.createTable(identity, new Table(
+                        databaseName,
+                        tableName,
+                        "admin",
+                        TableType.VIRTUAL_VIEW.toString(),
+                        storage,
+                        ImmutableList.of(),
+                        ImmutableList.of(),
+                        ImmutableMap.of(),
+                        Optional.of(originalViewText),
+                        Optional.of(expandedViewText)),
+                MetastoreUtil.buildInitialPrivilegeSet("admin"));
+    }
+
+    // The intent here is not to exhaustively test Coral itself, only that we have properly integrated with it. Queries of all different shapes are tested within Coral.
+    @Test
+    public void testSimpleCoralIntegration()
```

Comment:
We should move this to `io.prestosql.tests.hive.TestHiveViews` which already tests Hive view translation. Many of those tests will need to be updated anyway, as the translation will work with Coral.

Reply:
I have trouble running product test with following command "`presto-product-tests-launcher/bin/run-launcher test run --environment singlenode -t io.prestosql.tests.hive.TestHiveViews.testSelectOnView`", it complains about UNKNOWN HOST presto-master or cannot connect to presto-master.